database/seeds/BodySeeder.php

```diff
@@ -48,36 +29,324 @@ public function run()
             'name'          =>  'commission-european',
         ]);
 
+	Address::create([ //TODO: Validate address
+            'country_id'    =>  202,
+            'street'        =>  'Campus de Elviña',
+            'zipcode'       =>  '15071',
+            'city'          =>  'A Coruna',
+        ]);
+
     	Body::create([
-            'type_id'       =>  2,
+            'type_id'       =>  1,
             'address_id'    =>  1,
-    		'name'			=>	'AEGEE-Europe',
-            'email'         =>  'headoffice@aegee.org',
-            'legacy_key'    =>  'AEU' //No clue if this is actually correct.
+    	    'name'	    =>	'AEGEE-A Coruna',
+            'email'         =>  'aegee.coruna@gmail.com',
+	    'phone'	    =>  '',
+            'legacy_key'    =>  'TMP' //TODO
     	]);
+	
+        Address::create([
+            'country_id'    =>  81,
+            'street'        =>  'Templergraben 55',
+            'zipcode'       =>  '52056',
+            'city'          =>  'Aachen',
+        ]);
 
     	Body::create([
             'type_id'       =>  1,
             'address_id'    =>  2,
-    		'name'			=>	'AEGEE-Enschede',
-            'email'         =>  'headoffice@aegee.org',
-            'legacy_key'    =>  'ENS' //No clue if this is actually correct.
+    	    'name'	    =>	'AEGEE-Aachen',
+            'email'         =>  'info@aegee-aachen.org',
+	    'phone'	    =>  '+49 2418097121',
```

Comment:
This actually raises the question to me whether or not we should store country code along with bodies. I think it would be better to *exclude* the country code, as we can store the phone country code with each body and attach that only when necessary.
If you/anyone else agrees I suggest to change `'+49 2418097121'` into `'2418097121'` And same for all other bodies